packages/core/src/components/tag/tag.tsx

```diff
@@ -88,7 +88,7 @@ export class Tag extends React.PureComponent<ITagProps, {}> {
         ) : null;
 
         return (
-            <span {...htmlProps} className={tagClasses}>
+            <span {...htmlProps} className={tagClasses} tabIndex={interactive ? 0 : -1}>
```

Comment:
i think we want `undefined` in the false case (emphasis mine):

> A negative value (usually `tabindex="-1"`) means that the element **should be focusable, but should not be reachable** via sequential keyboard navigation. Mostly useful to create accessible widgets with JavaScript.
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex